File: VB/MyClass.vb
Imports System
Imports System.Linq
Imports System.Collections.Generic

Namespace NewProject
    Public Class MyDataClass
        Public Sub New()

        End Sub
        ' Fields...
        Private _Date As Date
        Private _Sales As Single
        Private _ID As Integer
        Private _Name As String
        Public Property ID() As Integer
            Get
                Return _ID
            End Get
            Set(ByVal value As Integer)
                _ID = value
            End Set
        End Property
        Public Property Name() As String
            Get
                Return _Name
            End Get
            Set(ByVal value As String)
                _Name = value
            End Set
        End Property
        Public Property Sales() As Single
            Get
                Return _Sales
            End Get
            Set(ByVal value As Single)
                _Sales = value
            End Set
        End Property
        Public Property [Date]() As Date
            Get
                Return _Date
            End Get
            Set(ByVal value As Date)
                _Date = value
            End Set
        End Property
    End Class
End Namespace

File: CS/MyClass.cs
using System;
using System.Linq;
using System.Collections.Generic;

namespace NewProject
{
    public class MyDataClass
    {
        public MyDataClass()
        {

        }
        // Fields...
        private DateTime _Date;
        private float _Sales;
        private int _ID;
        private string _Name;
        public int ID
        {
            get { return _ID; }
            set
            {
                _ID = value;
            }
        }
        public string Name
        {
            get { return _Name; }
            set
            {
                _Name = value;
            }
        }
        public float Sales
        {
            get { return _Sales; }
            set
            {
                _Sales = value;
            }
        }
        public DateTime Date
        {
            get { return _Date; }
            set { _Date = value; }
        }
    }
}
